python/paddle/fluid/tests/unittests/test_concat_op.py

```diff
@@ -338,21 +340,44 @@ def setUp(self):
         self.x = np.random.random(self.input_shape).astype("float32")
         self.place = fluid.CUDAPlace(0) \
             if fluid.is_compiled_with_cuda() else fluid.CPUPlace()
-        self.set_program()
 
-    def set_program(self):
-        self.program = fluid.Program()
-        with fluid.program_guard(self.program):
-            input = fluid.layers.assign(self.x)
-            tensor_array = fluid.layers.create_array(dtype='float32')
-            zero = fluid.layers.fill_constant(shape=[1], value=0, dtype="int64")
+    def set_program(self, use_paddle_api):
```

Comment:
`use_paddle_api` is confusing name. Maybe name it as `use_fluid_api` is better. Then `use_fluid_api = not use_paddle_api`.

I mentioned it because fluid API is also Paddle API.

Reply:
Thanks, done